INSTALL.md

```diff
@@ -49,6 +49,22 @@ Building F\* from sources section below) or at least in the latest
 [F\* binaries on GitHub]: https://github.com/FStarLang/FStar/releases
 [automatic weekly builds]: https://github.com/FStarLang/binaries/tree/master/weekly
 
+Binary builds come with binary builds of F* standard library. 
```

Comment:
Can we make it more explicit that this issue is only when extracting code to OCaml, not when just verifying programs.

Reply:
Added a section title to be extra explicit.